lua/nvim-navic/lib.lua

```diff
@@ -254,24 +254,15 @@ function M.request_symbol(for_buf, handler, client_id, file_uri)
 		}
 	end
 
-	vim.lsp.buf_request_all(
-		for_buf,
-		"textDocument/documentSymbol",
-		{ textDocument = textDocument_argument },
-		function(symbols)
-			if symbols[client_id] == nil then
-				return
-			elseif symbols[client_id].error ~= nil or symbols[client_id].result == nil then
-				vim.defer_fn(function()
```

Comment:
I understand! I see I removed the wrong check. The check for `if symbols[client_id] == nil` is not needed anymore since we are requesting only for a single client, and it will return something for that specific client, but the other two are of course needed. My bad 😅